files/max-pods-calculator.sh

```diff
@@ -140,12 +147,16 @@ else
     max_pods=$(calculate_max_ip_addresses_secondary_ips $enis_for_pods $INSTANCE_MAX_ENIS_IPS)
 fi
 
-# Limit the total number of pods that can be launched on any instance type based on the vCPUs on that instance type.
-MAX_POD_CEILING_FOR_LOW_CPU=110
-MAX_POD_CEILING_FOR_HIGH_CPU=250
-CPU_COUNT=$(echo $DESCRIBE_INSTANCES_RESULT | jq -r '.CpuCount' )
-if [ "$CPU_COUNT" -gt 30 ] ; then
-    echo $(min_number $MAX_POD_CEILING_FOR_HIGH_CPU $max_pods)
+if [ -n "$MAX_PODS_CEILING" ] ; then
```

Comment:
Apologies for not looking at this PR earlier.

Can we use a combination of [this PR](https://github.com/awslabs/amazon-eks-ami/pull/766/files) to achieve this? You can do the `min_number` logic on whatever invokes this script ?

Reply:
No worries @suket22 

I'm not sure what `SHOW_MAX_ALLOWED` would do here